app/controllers/ingredients_controller.rb

```diff
@@ -0,0 +1,5 @@
+class IngredientsController < ApplicationController
+  def index
+    @ingredients = Ingredients.all
```

Comment:
`Ingredients`? this is invalid because models should be in singular form `(Ingredient)`. I'm surprised that CI didn't detect this bug.

Reply:
You're right when it comes to models but this right here is a controller.

Reply:
🤔